include/cpp2util.h

```diff
@@ -2324,6 +2324,12 @@ struct args
     public:
         constexpr iterator(int c, char** v, int start) : argc{c}, argv{v}, curr{start} {}
 
+        using difference_type = std::ptrdiff_t;
+        using value_type      = std::string_view;
+
+        constexpr iterator() : argc(0), argv(nullptr), curr(0) {}
```

Comment:
Should this instead set initializers on the members and then use `= default`?

Reply:
Good call.  Done.